controllers/resource.go

```diff
@@ -68,5 +68,8 @@ func (r *OperatorPipelineReconciler) reconcileResources(meta metav1.ObjectMeta)
 		return err
 	}
 
+	r.secretExists(meta, "docker registry", "registry-dockerconfig-secret")
+	r.secretExists(meta, "github ssh", "github-ssh-credentials")
```

Comment:
We should stick with the existing pattern for these secrets as well. It probably would make sense after the refactor PR lands in main.

Reply:
What's the point of calling this if we are ignoring the return? Is there some kind of side effect happening in secretExists?

Reply:
however, i'll wait for the refactor.

Reply:
resolving because updated with refactored cod